samples/10.b.customization-password-input/README.md

```diff
@@ -25,308 +25,313 @@ A simple web page with a maximized Web Chat and hosted using React. This sample
+            <input
+               autoFocus={true}
+               type="password"
+            />
+         </label>
+      </form>
+   );
+};
+}
+```
 
-      this.handleCodeChange = this.handleCodeChange.bind(this);
-      this.handleSubmit = this.handleSubmit.bind(this);
+Next, build our CSS and apply class names to our component.
+
+```css
+.passwordInput {
+   background-color: Red;
+   border-radius: 3px;
+   color: White;
+   display: flex;
+   font-family: Calibri, 'Helvetica Neue', Arial, sans-serif;
+   margin: 10px;
+   padding: 5px;
+}
 
-      this.state = {
-         code: '',
-         submitted: false
-      };
-   }
+.passwordInput .passwordInput__box {
+   display: flex;
+   flex: 1;
+}
 
-   handleCodeChange({ target: { value: code } }) {
-      this.setState(() => ({ code }));
-   }
+.passwordInput .passwordInput__label {
+   padding: 10px;
+}
 
-   handleSubmit(event) {
-      event.preventDefault();
+.passwordInput .passwordInput__input {
+   border: 0px;
+   border-radius: 3px;
+   flex: 1;
+   letter-spacing: .5em;
+   outline: 0;
+   padding: 0 10px;
+   width: 100%;
+}
 
-      this.props.sendPostBack({ code: this.state.code });
-      this.setState(() => ({ submitted: true }));
-   }
+.passwordInput .passwordInput__input:disabled {
+   background-color: rgba(255, 255, 255, .5);
+   color: White;
+}
+```
 
-   render() {
-      const {
-         state: { code, submitted }
-      } = this;
+Then, apply the style sheet to our React component.
+
+```diff
+   return (
+-     <form>
++     <form className="passwordInput">
+-        <label>
++        <label className="passwordInput__box">
+-           <span>Please input your 2FA code</span>
++           <span className="passwordInput__label">Please input your 2FA code</span>
+            <input
+               autoFocus={true}
++              className="passwordInput__input"
+               type="password"
+            />
+         </label>
+      </form>
+   );
+```
+
+Then, add business logic to the component:
+
+- When the user type on the password box, save the code.
+- When the user press <kbd>ENTER</kbd> on the password box, submit the code and disable the password box.
+
+This component will have a state object with `code` and `submitted` values, initially set to an empty string and `false`, respectively. `code` will store the  code, and `submitted` will store a boolean value to indicate whether the code has been submitted or not. If the code has been submitted, disable the input box to prevent resubmission.
+
+This component will have two callback functions, called `handleCodeChange` and `handleSubmit`. The former will be called when the code has changed, it will save the value into `code` state. The latter will be called when the form is submitted (user press <kbd>ENTER</kbd> on the password box). It will submit the code via `sendPostBack`, and disable itself to prevent resubmission by updating `submitted` to `true`.
```

Comment:
```suggestion
This component will have two callback functions, called `handleCodeChange` and `handleSubmit`. The former will be called when the code has changed, and it will save the value into the `code` state. The latter will be called when the form is submitted (user press <kbd>ENTER</kbd> on the password box). It will submit the code via `sendPostBack`, and disable itself to prevent resubmission by updating `submitted` to `true`.
```